docs/UserGuide.md

```diff
@@ -87,22 +87,19 @@ Shows a list of all persons in the address book.
 
 Format: `list`
 
-### Editing a person : `edit`
+### Editing a resident : `edit`
 
-Edits an existing person in the address book.
+Edits an existing resident in the Hall-y.
```

Comment:
Should it be just 'in Hall-y', instead of 'in _the_ Hall-y'?

Reply:
Fixed, thanks.